modules/ROOT/pages/prereq-aws-terraform.adoc

```diff
@@ -62,6 +62,7 @@ ssh -i 'anypoint.pem' ec2-user@18.221.155.87
 | AWS_KEY_NAME | Specifies the AWS SSH key. Do not include the `.pem` extension.
 | AWS_REGION | Specifies the AWS region where Terraform creates the cluster, for example, `us-east-2`. This value must have the same value as AWS_DEFAULT_REGION.
 | AWS_DEFAULT_REGION | Must be the same value as AWS_REGION.
+| AWS_SESSION_TOKEN | Specifies AWS temporary security credentials.
```

Comment:
Let's move up this line right after `AWS_SECRET_ACCESS_KEY` and for the description let's change it to:
`Specifies the AWS temporary session token if any`

Reply:
Got it - thank you!